lib/fizzy/parser_expr.cpp

```diff
@@ -94,6 +94,14 @@ parser_result<Code> parse_expr(const uint8_t* pos)
         case Instr::i64_extend_i32_u:
             break;
 
+        case Instr::loop:
+        {
+            const auto type = *pos++;
```

Comment:
I think this should be statically typed for clarity: `const uint8_t type = *pos++;`